components/Faq.vue

```diff
@@ -69,33 +36,97 @@ export default {
       required: true
     }
   },
-  computed: {
-    sortedItems() {
-      return [...this.items].sort((a, b) => {
-        if (a.question === 'What is a hackathon?') {
-          return -1
-        } else if (b.question === 'What is a hackathon?') {
-          return 1
-        } else {
-          return 0
-        }
-      })
+  data() {
+    return {
+      leftFaqs: this.getFaqColumn(true),
+      rightFaqs: this.getFaqColumn(false)
     }
   },
   methods: {
-    openSesame: (e) => {
+    toggleBox: (e) => {
       e.target.classList.toggle('active')
       const panel = e.target.nextElementSibling
       if (panel.style.display === 'block') {
         panel.style.display = 'none'
       } else {
         panel.style.display = 'block'
       }
+    },
+    getFaqColumn(left) {
+      const isLeft = (e, i) => left ? i % 2 === 0 : i % 2 !== 0
```

Comment:
Do you need to pass in `e` here because it's not being used.

Reply:
Good observation, but we need it since it's a method that gets passed down to `.filter`, and i is the second variable.

Reply:
Oh, my bad I didn't realize that below in `filterFaq` i was actually supposed to be item not index.